vector/src/main/java/im/vector/app/features/home/room/detail/timeline/helper/TimelineDisplayableEvents.kt

```diff
@@ -51,7 +51,7 @@ object TimelineDisplayableEvents {
             EventType.STATE_ROOM_JOIN_RULES,
             EventType.KEY_VERIFICATION_DONE,
             EventType.KEY_VERIFICATION_CANCEL,
-    ) + EventType.POLL_START + EventType.STATE_ROOM_BEACON_INFO
+    ) + EventType.POLL_START + EventType.STATE_ROOM_BEACON_INFO + EventType.BEACON_LOCATION_DATA
```

Comment:
Why we added location data as displayable?

Reply:
I added it because we want the redacted events to appear in the timeline (there are still hidden in the general case). I aligned the behavior on what is done in web. I think this is good to see the redacted events to inform the user the location events were redacted.